vnext/ReactWindowsCore/DevSettings.h

```diff
@@ -102,6 +109,11 @@ struct DevSettings {
   // Until the ABI story is addressed we'll use this instead of the above for
   // the purposes of selecting a JSI Runtime to use.
   JSIEngineOverride jsiEngineOverride{JSIEngineOverride::Default};
+
+  /// Used to allow ChakraExecutor to query the instance creator for
+  /// bytecode/versioning information from scriptURL. Superseded by
+  /// PreparedScriptStore on the JSI stack, and will be removed soon.
```

Comment:
>removed soon [](start = 56, length = 12)

Please file a work item for this, and mention it in the code.

Reply:
I was going to ask if we were filing items on Github itself or doing some sort of VSTS syncing, but it looks like Yichen pulled things up on Github so that's the right place?

---
In reply to: [342724588](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342724588) [](ancestors = 342724588)

Reply:
I always put issues on GitHub if I can. This is an open-sourced repo after all - if we are aware of an issue, we should let the community know.

---
In reply to: [342860895](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342860895) [](ancestors = 342860895,342724588)

Reply:
A bit tangential, but I think there was some bot somewhere that could synchronize issues between Github and VSTS. Might be worth looking into just to consolidate things.

---
In reply to: [342862150](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342862150) [](ancestors = 342862150,342860895,342724588)